app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -82,6 +86,54 @@ const ClaimScreen: React.FC = () => {
     }
   };
 
+  const verifyEmail = async (email: string) => {
+    const armoredKey = await pgpLocalStorage.get();
+    try {
+      const uploadResponse = await axios.post<UploadPGPKeyResponse>(
+        "https://keys.openpgp.org/vks/v1/upload",
```

Comment:
Can we put url into config file same as ritestream?

Reply:
Im happy for this to be hard coded for v 0.1